collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# start off by doing a gather, which includes gather_network_logs_basics.
+/usr/bin/gather
+
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/network_logs"}
+OVNK_EXTRAS_STORE_PATH=${OUT:-"ovnk_extras_store"}
+
+mkdir -p "${NETWORK_LOG_PATH}"/
+
+function gather_extra_ovn_kubernetes_data_interconnect_mode {
+  echo "INFO: Gathering ovn-kubernetes extra logs"
+  OVNKUBE_CONTROLLER_PODS=($(oc -n openshift-ovn-kubernetes get pods -l app=ovnkube-node -o=jsonpath='{.items[*].metadata.name}'))
+  # collect additional ovnkube logs from each node
+  CONTAINERS="ovnkube-controller"
+  for CONTAINER in "${CONTAINERS}" ; do
```

Comment:
do we need this loop?

Reply:
heh, we do not at all. I was trying to think ahead if we would ever care for other containers in that pod.
But less is more; we can add it should the need arise.